PetGame/Controllers/LeaderboardController.cs

```diff
@@ -5,85 +5,63 @@
 
 namespace PetGame
 {
+    public class LeaderboardEntry
+    {
+        public string PetName { get; set; }
+        public int score { get; set; }
+        public string OwnerName { get; set; }
+
+        public LeaderboardEntry(string nPetName, int nScore, string nOwnerName)
+        {
+            PetName = nPetName;
+            score = nScore;
+            OwnerName = nOwnerName;
+        }
+    }
+
     [Route("api/[controller]")]
     public class LeaderboardController : Controller
     {
         //initialize connection to SQL Server
         private readonly SqlManager sqlManager;
 
-        public LeaderboardController(SqlManager sqlManager) : base()
+        public LeaderboardController(SqlManager sqlManager)
         {
             this.sqlManager = sqlManager;
         }
 
         [HttpGet]
-        public IActionResult Get()
+        public IActionResult Get(int NumResults = 10)
```

Comment:
How is this variable set? You don't specify attributes to indicate that it's part of the request body or not. In addition, this could use xmldoc comments (`///` comments)